src/main/target/STM32_UNIFIED/target.h

```diff
@@ -239,6 +239,13 @@
 #define USE_ACCGYRO_LSM6DSO
 #define USE_ACCGYRO_BMI270
 
+#if defined(STM32F405) || (STM32H743)
```

Comment:
Does this work as intended? I believe this should be ```#if defined(STM32F405) || defined(STM32H743)```

Reply:
https://stackoverflow.com/questions/9946108/if-defined-x-y-is-this-valid

Reply:
Haven't noticed. You are probably right.

Reply:
Doesn't matter if it works or not. We should be clear and consistent.

Reply:
I read it as "if STM32F405 is defined or STM32H743 is true". If STM32H743 is just defined this shouldn't even compile. I'm not an expert on this preprocessor stuff so I could be wrong.

Reply:
Although it could be valid as commented in the article above it's confusing to read as I agree with your interpretation.